### PR TITLE
Wrap extended type_string call

### PR DESCRIPTION
**Pull request by mloviska**
Follow up PR of [#6792](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6792)

- Related ticket: [poo#46679](https://progress.opensuse.org/issues/46679)

### CODE REVIEW

File: tests/yast2_gui/yast2_hostnames.pm [hunk 1]
```diff
@@ -21,10 +21,18 @@ use testapi;
 use utils 'clear_console';
 use x11utils qw(turn_off_kde_screensaver turn_off_gnome_screensaver);
 
+
+# Enable additional arguments for nested calls of wait_still_screen
+# SLOW_TYPING_SPEED == max_interval => 13
+sub type_string_extended {
```
Comment (rwx788) [inline tests/yast2_gui/yast2_hostnames.pm]: What's about moving it to some lib, for example to `utils` where `type_string_slow` is defined. I've suggested extracting this method, so we can reuse it not only in this test module. For single module we could have put parameters to the hash and simply reuse it all 3 calls.

Comment (mloviska) [reply tests/yast2_gui/yast2_hostnames.pm]: Well, I thought that we have agreed on leaving it for now, since the parameters might not be suitable for osd or o3. Nevertheless I will extract it

Comment (rwx788) [reply tests/yast2_gui/yast2_hostnames.pm]: Yeah, that's why I've merged PR with fix to yast2_hostnames. But you've created this PR, so I thought we address it now then =).

File: tests/yast2_gui/yast2_hostnames.pm [hunk 2]
```diff
@@ -18,25 +18,20 @@ use base "y2x11test";
...
     my $hosts_params = {
         ip    => '195.135.221.134',
         fqdn  => 'download.opensuse.org',
         alias => 'download-srv'
     };
-    my %execute = (
-        gnome => \&turn_off_gnome_screensaver,
-        kde   => \&turn_off_kde_screensaver
-    );
-    $execute{$dm}->();
```
Comment (rwx788) [inline tests/yast2_gui/yast2_hostnames.pm]: Is this change on purpose? If yes, we can remove `$dm` initialization too.

Comment (mloviska) [reply tests/yast2_gui/yast2_hostnames.pm]: yup, thanks!
